src/environment/stella_environment.cpp

```diff
@@ -109,7 +107,7 @@ void StellaEnvironment::reset() {
   int noopSteps;
   noopSteps = 60;
 
-  emulate(PLAYER_A_NOOP, PLAYER_B_NOOP, noopSteps);
+  emulate(PLAYER_A_NOOP, PLAYER_B_NOOP, 0.0, 0.0, noopSteps);
```

Comment:
minor nit: should these be `1.0` to match the default parameter values?

same comment for all the `0.0`s below

Reply:
Good catch, thanks!